db/seeds.rb

```diff
@@ -9,4 +9,43 @@
 # See https://github.com/rails/rails/issues/35812
 ActiveJob::Base.queue_adapter = Rails.application.config.active_job.queue_adapter
 
-Run.create(s3_filename: SecureRandom.uuid)
+Game.create(name: 'Assassin\'s Creed')
```

Comment:
Is there a reason to add a bunch of SRDC-less games in the seed file?

Reply:
Oops, I had populated the SRDC games at the command line after and forgot to add them here.